src/System.Linq.Expressions/tests/BinaryOperators/Assignment/OpAssign.cs

```diff
@@ -224,7 +224,7 @@ public void ThrowIfNoSuchBinaryOperation(MethodInfo assign)
             Assert.Throws<InvalidOperationException>(() => withAssignment(variable, value));
         }
 
-        private static IEnumerable<object[]> AssignmentMethods()
+        public static IEnumerable<object[]> AssignmentMethods()
```

Comment:
Strangely, I started to get errors with those private members out of the blue today (basically it complains it can't find a *public* member as specified in MemberDataAttribute). Maybe I somehow pulled in a newer version of Xunit? I can omit those changes from the PR.

@JonHanna - Any clue? Have you seen this before?